asv_bench/benchmarks/utils/data_shapes.py

```diff
@@ -166,6 +166,9 @@
     ),
 ]
 DEFAULT_CONFIG = {}
+DEFAULT_CONFIG["MergeCategoricals"] = (
+    [[10_000, 2]] if ASV_DATASET_SIZE == "big" else [[1_000, 2]]
```

Comment:
Why don't we want to use binary_op data size as for TimeMerge? Also, why only 2 colums, not 10, for example?

Reply:
As for the size, it turned out to be not very convenient and fast to fit each operation to the same size. In order to save time, we can not do this. This size is enough to try to optimize this function. I tested this on my local machine. Maybe we can only increase the number of rows, since the local machine has 8 cores, and the benchmark machine has 44 cores. For example: `[100_000, 2]`.

To vary the number of columns, we will need to write a special function that generates a dataframe with the required data type. This is more complicated than just transferring the creation of the dataframe as is. Again, this size is enough to try to optimize this function I think.